packages/front-end/components/Experiment/Templates/TemplatesPage.tsx

```diff
@@ -0,0 +1,214 @@
+  const canCreate = permissionsUtil.canCreateExperimentTemplate({
+    project: project,
+  });
+  const canEdit = (templ: ExperimentTemplateInterface) =>
+    permissionsUtil.canUpdateExperimentTemplate(templ, {});
+  const canDelete = (templ: ExperimentTemplateInterface) =>
+    permissionsUtil.canDeleteExperimentTemplate(templ);
+
+  const flattenedTemplates = useAddComputedFields(
+    allTemplates,
+    (templ) => {
+      return {
+        ...omit(allTemplates, ["templateMetadata"]),
+        templateName: templ.templateMetadata.name,
+        templateDescription: templ.templateMetadata.description,
+        usage: templateExperimentMap[templ.id]?.length ?? 0,
+      };
+    },
+    []
+  );
+
+  const { items, SortableTH } = useSearch({
+    items: flattenedTemplates,
+    defaultSortField: "templateName",
+    localStorageKey: "templates",
+    searchFields: ["templateName^3", "tags", "templateDescription"],
+  });
+
+  const hasTemplates = items.length > 0;
+  const showProjectColumn = items.some((t) => !!t.project);
+
+  if (loading || !ready) {
+    return <LoadingOverlay />;
+  }
+
+  if (error) {
+    return <div className="alert alert-danger">{error.message}</div>;
+  }
+
+  return hasTemplates ? (
+    <Box>
+      <table className="appbox table gbtable table-hover">
+        <thead>
+          <tr>
+            <SortableTH field="templateName">Template Name</SortableTH>
+            <SortableTH field="templateDescription">Description</SortableTH>
+            <SortableTH field="tags">Tags</SortableTH>
+            {showProjectColumn && (
+              <SortableTH field="project">Project</SortableTH>
+            )}
+            <SortableTH field="dateCreated">Created</SortableTH>
+            <SortableTH field="usage">Usage</SortableTH>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          {items.map((t) => {
+            const templateUsage = t.usage;
+            return (
+              <tr key={t.id} className="hover-highlight">
+                <td data-title="Template Name" className="col-2">
+                  {templateUsage ? (
+                    <Link href={`/experiments/template/${t.id}`}>
+                      {t.templateName}
+                    </Link>
+                  ) : (
+                    <Tooltip content="This template hasn’t been used to create any experiments yet">
+                      <Text>{t.templateName}</Text>
+                    </Tooltip>
+                  )}
+                </td>
+                <td data-title="Description" className="col-3">
+                  {t.templateDescription}
+                </td>
+                <td data-title="Tags">
+                  <SortedTags
+                    tags={Object.values(t.tags ?? [])}
+                    useFlex={true}
+                  />
+                </td>
+                <td className="text-gray col-2">
+                  {t.project ? getProjectById(t.project)?.name : ""}
+                </td>
```

Comment:
We're conditionally rendering this column only if `showProjectColumn` is true, so we should do the same here.